parser/src/test/scala/jawn/SyntaxCheck.scala

```diff
@@ -92,7 +92,15 @@ class SyntaxCheck extends Properties("SyntaxCheck") {
   }
 
   property("invalid unicode is invalid") = {
-    isValidSyntax("\"\\uqqqq\"") != true
+    isValidSyntax("\"\\uqqqq\"") != true &&
+    isValidSyntax("\"\\ughij\"") != true &&
```

Comment:
Would you mind adding tests for `"\uGGGG"` and `"\ugggg"` here, just to verify that the boundary is exactly right for both lower and uppercase?